packages/manager/modules/anthos/src/dashboard/general-information/order-host/order-host.routing.js

```diff
@@ -14,5 +14,8 @@ export default /* @ngInject */ ($stateProvider) => {
         return 'order-host';
       },
     },
+    atInternet: {
+      rename: 'hpc::anthos::project::order-host',
```

Comment:
may be you extract common, duplicated prefix "hpc::anthos::project::" into constants or route file?

Reply:
This is done. Thanks